File: HELLO_PYTHON/day06/mysql_update.py
import pymysql
from sqlalchemy.dialects.mysql.pymysql import MySQLDialect_pymysql
 
# MySQL에 연결
conn = pymysql.connect(host='localhost', user='root', password='python',
                       db='python', port=3305, charset='utf8')

e_id = "5"
e_name = "6"
gen = "6"
addr = "6"

curs = conn.cursor()

# 수정된 SQL 쿼리에는 각 변수 주변에 따옴표(')를 추가했습니다.
sql = f"""
    update emp
    set
        e_name='{e_name}',
        gen='{gen}',
        addr='{addr}'
    where 
        e_id='{e_id}'    
"""


cnt = curs.execute(sql)
print(cnt)
#print(curs.rowcount)


conn.commit()

curs.close()
conn.close()             

File: HELLO_NODE2/mysql_update.js
const mysql      = require('mysql'); //import 역할
const connection = mysql.createConnection({
  host     : 'localhost',
  user     : 'root',
  password : 'python',
  database : 'python',
  port : 3305 //기본 포트가 3306이므로 포트 지정도 해주어야 한다.
});


connection.connect(function(err) {
  console.log("start");
  var sql = "update emp set e_name = '2', gen = '3' where e_id = '1' ";
  connection.query(sql, function (err, result) {
    console.log("1 record updated");
	console.log(result.affectedRows);
  });
});

 console.log("end");